frontend/src/metabase/lib/api.js

```diff
@@ -289,7 +290,11 @@ export class Api extends EventEmitter {
 
           if (status >= 200 && status <= 299) {
             if (options.transformResponse) {
-              body = options.transformResponse(body, { data });
+              body = options.transformResponse({
+                body,
+                data,
+                response: unreadResponse,
```

Comment:
Since I need to pass a new key, I think it's a good time to refactor `transformResponse` API, although nobody uses this API outside this PR.